2020/05/22 15:48:53 [error] 6#6: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.192.1, server: soapboxtest.com, request: "GET /users HTTP/1.1", upstream: "fastcgi://192.168.192.2:9000", host: "soapboxtest.com"
2020/05/22 15:48:58 [error] 6#6: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.192.1, server: soapboxtest.com, request: "GET /users HTTP/1.1", upstream: "fastcgi://192.168.192.2:9000", host: "soapboxtest.com"
2020/05/22 15:48:59 [error] 6#6: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.192.1, server: soapboxtest.com, request: "GET /users HTTP/1.1", upstream: "fastcgi://192.168.192.2:9000", host: "soapboxtest.com"
2020/05/22 15:48:59 [error] 6#6: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.192.1, server: soapboxtest.com, request: "GET /users HTTP/1.1", upstream: "fastcgi://192.168.192.2:9000", host: "soapboxtest.com"
